src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -1810,127 +1810,70 @@ namespace {
 
         int numrecords = keyword.size();
 
+        Tuning tuning(m_tuning.get(currentStep > 0 ? m_currentStep - 1 : 0));
```

Comment:
Part of the logic in `DynamicState` class is to avoid this `if step > 0` bla bla bla - this *should* work:
```
Tuning tuning(m_tuning.get(currentStep));
```

Reply:
fixed.